cluster/gce/config-test.sh

```diff
@@ -44,10 +44,10 @@ WINDOWS_NODE_LABELS=${WINDOWS_NODE_LABELS:-}
 NODE_LOCAL_SSDS_EPHEMERAL=${NODE_LOCAL_SSDS_EPHEMERAL:-}
 # Turning GRPC based Konnectivity testing on id advance of
 # removing the SSHTunnel code.
-export KUBE_ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE=true
-export PREPARE_KONNECTIVITY_SERVICE="${KUBE_ENABLE_KONNECTIVITY_SERVICE:-true}"
-export EGRESS_VIA_KONNECTIVITY="${KUBE_ENABLE_KONNECTIVITY_SERVICE:-true}"
-export RUN_KONNECTIVITY_PODS="${KUBE_ENABLE_KONNECTIVITY_SERVICE:-true}"
+export KUBE_ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE="${KUBE_ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE:-false}"
```

Comment:
CI should opt into this in appropriate jobs.
I'm not clear why we added this _and_ `EGRESS_VIA_KONNECTIVITY` yet, while this one was not overridable.

Reply:
I have mixed feelings on this... this has to be a reasonable default configuration before 1.22 is released, so enabling by default makes sense to me. That said, https://github.com/kubernetes/kubernetes/issues/102904 clearly indicates there's an issue to resolve, so I think it is reasonable to back out the "on by default" change until that is resolved.

Reply:
I think that it stills need to mature a bit more for being default https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/239